timely_beliefs/visualization/utils.py

```diff
@@ -146,42 +151,41 @@ def plot(
         )
 
 
-def timedelta_to_human_range(
-    df: "classes.BeliefsDataFrame"
-) -> Tuple["classes.BeliefsDataFrame", str]:
-    timedelta_span = max(df["belief_horizon"]) - min(df["belief_horizon"])
+def timedelta_to_human_range(s: pd.Series) -> Tuple[pd.Series, str]:
+    """Convert a pandas Series of timedeltas to a pandas Series of floats,
+    and extract a time unit that gives a nice human readable range of floats.
+    """
```

Comment:
Can you give one example in the docstring? It would help to see right away what you mean with "readable range of floats". That term is weird.

Reply:
I added an example. Is that adequate, or should I comment on how the time unit is chosen?

Reply:
Great!. Well, the docstring says "extract a time unit" and I did not know what that means. Now I see it sets a time unit as a string on the series object.The series' type is now float64, so that "hours" string is set as what  on the series object?

Reply:
The string is not set on the Series object, but simply the second entry of the returned tuple.